frontend/src/components/Header/UserInfo.js

```diff
@@ -1,9 +1,12 @@
-import React from 'react';
+import React  from 'react';
+import { useHistory } from 'react-router-dom';
 import authUtils from '../AuthUtils';
 
 import '../../styles/header.css';
 import Button from 'react-bootstrap/Button';
 
+import { LANDING } from '../../constants/routes.js';
+
```

Comment:
What's the specific style used for these imports?

Reply:
As described in #76, I think we've settled on:
```
React stuff

Firestore, Bootstrap, other libraries

Local imports
```
I've updated the spacing to reflect this.